engine-rest/engine-rest-openapi/src/main/templates/paths/task/{id}/deployed-form/get.ftl

```diff
@@ -0,0 +1,76 @@
+{
+
+  <@lib.endpointInfo
+      id = "getDeployedForm"
+      tag = "Task"
+      desc = "Retrieves the deployed form that is referenced from a given task. For further
+            information please refer to the
+            [User Guide](${docsUrl}/user-guide/task-forms/#embedded-task-forms)." />
+
+  "parameters" : [
+
+    <@lib.parameter
+        name = "id"
+        location = "path"
+        type = "string"
+        required = true
+        last = true
+        desc = "The id of the task to get the deployed form for."/>
+
+  ],
+
+  "responses" : {
+
+    <@lib.response
+        code = "200"
+        mediaType = "application/xhtml+xml"
```

Comment:
the `ExceptionDto` is still referenced in response by default, we should fix the macro